autopilot/Lab_2/wifi_server.py

```diff
@@ -1,104 +1,66 @@
 import socket
 import picar_4wd as fc
 import threading
-import subprocess
-import json
-#from picamera import PiCamera
 import time
-import io
-import struct
-import numpy as np
-import cv2 as cv
+import json
+
+HOST = "192.168.0.14"  # IP address of your Raspberry PI
+PORT = 65432           # Port to listen on (non-privileged ports are > 1023)
+POWER = 10             # Power of motors
 
```

Comment:
why was this removed?

Reply:
im not sure why its showing green tbh, didnt touch that, maybe its the formatting

Reply:
oh nvm, you're pointing to the cv2 and stuff, yeah i removed it temporary to test sending basic metrics in threads, will re-add the video feed stuff after